### ISSUE TITLE
Notification autoclose does not appear to work

### ISSUE THREAD

**Issue by iotaweb** [zurb/foundation-apps#532]
Hello, 

The **autoclose** property doesn't appear to work for notifications in my application. I think the reason is that `scope.active` is not set  to true until after the 50 millisecond timeout runs, i.e.
```
        scope.active = false;
        ...
        //due to dynamic insertion of DOM, we need to wait for it to show up and get working!
        setTimeout(function() {
          scope.active = true;
          foundationApi.animate(element, scope.active, animationIn, animationOut);
        }, 50);
```
So when the following code block runs, `scope.active` is still still set to `false`, e.g.
```          
        // close if autoclose
        if (scope.autoclose && scope.active) {
          setTimeout(function() {
            scope.hide();
          }, parseInt(scope.autoclose));
        };
```
However, I can get notifications to autoclose if I put the above block in the `setTimeout` function, e.g.
```
        //due to dynamic insertion of DOM, we need to wait for it to show up and get working!
        setTimeout(function() {
          scope.active = true;
          foundationApi.animate(element, scope.active, animationIn, animationOut);
          // close if autoclose
          if (scope.autoclose && scope.active) {
            setTimeout(function() {
              scope.hide();
            }, parseInt(scope.autoclose));
          };          
        }, 50);
```
Not sure if this is the best fix, but it now works for me!

Regards, Rob

**Comment by iotaweb** [comment]
Actually, this may have been fixed - I was pulling from bower ~1.0.x which resolved to 1.0.3. It seems to work if I pull from the latest on #master. Please feel free to close if you're happy it has been addressed. Tx.

**Comment by gakimball** [comment]
Yep, #483 fixes this I believe. We're trying to put out a 1.1 tomorrow, so stay tuned!
